src/datepicker/datepicker.component.ts

```diff
@@ -120,20 +120,38 @@ export class DatePicker implements OnDestroy {
 
 	@Input() skeleton = false;
 
+	@Input() plugins = [];
+
+	@Input()
+	set flatpickrOptions(options: FlatpickrOptions) {
+		this._flatpickrOptions = Object.assign({}, this._flatpickrOptions, options);
+	}
+	get flatpickrOptions(): FlatpickrOptions {
+		const plugins = [...this.plugins];
+		if (this.range) {
+			plugins.push(rangePlugin({ input: "#" + this.id + "-rangeInput"}));
+		}
+		return Object.assign({}, this._flatpickrOptions, this.flatpickrBaseOptions, {plugins});
+	}
+
+	@Input()
+	set flatpickrOptionsRange (options: FlatpickrOptions) {
```

Comment:
Didn't we talk about deprecating range and just using `flatpickrOptions` with `range` controlling the switch between the versions?

Everything else seems correct except `flatpickrOptionsRange` became an input except getting deprecated.

Reply:
lol yeah that shouldn't be an `Input` ... the rest of the implementation is valid I think though